src/main/java/com/nordstrom/kafka/connect/sqs/SqsConnectorConfigKeys.java

```diff
@@ -26,6 +26,8 @@ public enum SqsConnectorConfigKeys {
   TOPICS("topics"),
   SQS_REGION("sqs.region"),
   SQS_ENDPOINT_URL("sqs.endpoint.url"),
+  SQS_MESSAGE_ATTRIBUTES_ENABLED("sqs.message.attributes.enabled"),
+  SQS_MESSAGE_ATTRIBUTES_LIST("sqs.message.attributes.list"),
```

Comment:
A name adjustment will add clarity and allow for additional behaviors here, such as "excluding".

```suggestion
  SQS_MESSAGE_ATTRIBUTES_INCLUDE_LIST("sqs.message.attributes.include.list"),
```